### ISSUE TITLE
conflict with symfony/console v5.0.1

### ISSUE THREAD

**Issue by cofirazak** [fetzi/phpspec-watcher#5]
After composer require fetzi/phpspec-watcher --dev
there is a conflict with my already installed symfony/console v5.0.1

Because fetzi/phpspec-watcher v1.0.0 requires symfony/process ^3.0
but symfony/process ^3.0 conflict with symfony/console[v5.0.1].

Log:
```
composer require fetzi/phpspec-watcher --dev                                                                                          
Using version ^1.0 for fetzi/phpspec-watcher
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install fetzi/phpspec-watcher 1.0.2
    - Conclusion: don't install fetzi/phpspec-watcher v1.0.1
    - Conclusion: remove symfony/console v5.0.1
    - Installation request for fetzi/phpspec-watcher ^1.0 -> satisfiable by fetzi/phpspec-watcher[1.0.2, v1.0.0, v1.0.1].
    - Conclusion: don't install symfony/console v5.0.1
    - fetzi/phpspec-watcher v1.0.0 requires symfony/process ^3.0 ->...
```

My temporary solution was to create and use this gist: 
https://gist.github.com/cofirazak/8dcf9b5204c7bdafd0f4bcff636a55ee

**Comment by fetzi** [comment]
@cofirazak your problem should be fixed in version 2.0.0.

**Comment by cofirazak** [comment]
> @cofirazak your problem should be fixed in version 2.0.0.

@fetzi thanks for reply!
